coalahtml/coala_html.py

```diff
@@ -35,7 +35,7 @@ def main():
 
     dir_path = create_dir(os.path.abspath(args.dir))
 
-    if len(list(os.walk(dir_path))) > 0:
+    if len(list(os.walk(dir_path))) > 0:  # pragma: no cover
```

Comment:
^ Anyway this is going to be in https://github.com/coala-analyzer/coala-html/pull/81/commits/58f9d34bb324d1885b618d53dc406dd22ad221d4